src/components/Header/components/WalletIcon/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { makeStyles } from '@material-ui/core/styles'
+import * as React from 'react'
+
+import Col from 'src/components/layout/Col'
+import Img from 'src/components/layout/Img'
+import WALLET_ICONS from './icons'
+
+const useStyles = makeStyles({
+  container: {
+    marginLeft: '5px',
+    marginRight: '10px',
+    letterSpacing: '-0.5px',
+  },
+  icon: {
+    maxWidth: 'none',
+  },
+})
+
+interface WalletIconProps {
+  provider: string
+}
+
+const WalletIcon: React.FC<WalletIconProps> = ({ provider }) => {
```

Comment:
Let's use `React.ReactElement` type here, because React.FC has its drawbacks, for example it will automatically add children type.

Read more: https://github.com/facebook/create-react-app/pull/8177